governance

```diff
@@ -0,0 +1,52 @@
+
+- Reporting bugs and/or fixing them. 
+- posting feature requests
+- taking an ownership on maintineg spcific packages up to date. 
+- updating/writing documentation
+- submitting pull requests or patches
+- Improving the Flatcar website.
+- Joining discussions on Twitter, our community [Slack channel](https://kubernetes.slack.com/archives/C03GQ8B5XNJ) ,[Matrix channel](https://app.element.io/#/room/#flatcar:matrix.org) and office hours. 
+- Evangelizing about the project. 
+
+For first-time contributors, it’s recommended to start by going through [Contribuiting-git](https://github.com/flatcar-linux/Flatcar/blob/main/contributions-git.md), and joining our community Slack channel or Matrix channel. 
+
+
+### Committers
+
+Committers are active community members who are responsible for the overall quality and stewardship of the project. Committers are part of the organization with write access to all repositories. Committers are expected to remain actively involved in the project and participate in voting and discussing of proposed project level changes.
+
+Typically, a potential committer needs to show that they have a sufficient understanding of the project, its objectives, and its strategy. To become a committer, you are expected to:
+- express an intresset to the exsiting maintainers about becoming cecoming a coomiter.
+- Have creted substentials PRs
+
+
+### Maintainers
+
+Contribution per-se does not qulified to become a maintainer. Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues.
+Maintainership is about building trust with the current maintainers of the project and being a person that they can depend on to make decisions in the best interest of the project in a consistent manner.
+
+Committers wanting to become maintainers are expected to:
+
+- Collaborate well.
+- Demonstrate a deep and comprehensive understanding of Fltcar code base, technical goals, and directions.
+- Actively engage with major Chaos Mesh feature proposals and implementations.
+
+Periodically, the existing maintainers curate a list of contributors that have shown regular activity on the project over the prior months. The nominating maintainer will create a PR to update the Maintainers List. It is recommended to describe the reasons for the nomination and the contribution of the nominee in the PR. Upon consensus of incumbent maintainers, the PR will be approved and the new maintainer becomes active.
```

Comment:
delete this para? I don't think we do this.

Reply:
I want to also lay the foundation for growth and establish a clear path for becoming a maintainer.

Reply:
@ahrkrak I think we should keep it, any objection?